etc/adminhtml/di.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <virtualType name="Magento\Catalog\Ui\DataProvider\Product\Form\Modifier\Pool">
```

Comment:
@ryanreeves-taxjar Do we need a `type` attribute on this tag as well?

Reply:
Core uses this virtualType with and without the "type" attribute, but I don't see any reason not to add it.